examples/electron/electron.js

```diff
@@ -26,11 +26,10 @@ function createWindow() {
 		console.log("Message received: " + message.symbol);
     });
 
-    /*
-    mainWindow.on("closed", function () {
+    mainWindow.addListener("closed", function () {
```

Comment:
Mimic behavior of OpenFin within the Electron example by closing the application when the "main" window is closed.